lib/models/LifecycleConfiguration.js

```diff
@@ -648,6 +648,62 @@ class LifecycleConfiguration {
             });
         });
     }
+
+    /**
+     * Get XML representation of lifecycle configuration object
+     * @param {object} config - Lifecycle configuration object
+     * @return {string} - XML representation of config
+     */
+    static getConfigXml(config) {
+        const rules = config.rules;
+        const rulesXML = rules.map(rule => {
+            const { ruleID, ruleStatus, filter, actions } = rule;
+            const ID = `<ID>${ruleID}</ID>`;
+            const Status = `<Status>${ruleStatus}</Status>`;
+
+            const { rulePrefix, tags } = filter;
+            const Prefix = rulePrefix ? `<Prefix>${rulePrefix}</Prefix>` : '';
+            let tagXML = '';
+            if (tags) {
+                tagXML = tags.map(t => {
+                    const { key, val } = t;
+                    const Tag = `<Tag><Key>${key}</Key>` +
+                        `<Value>${val}</Value></Tag>`;
+                    return Tag;
+                }).join('');
+            }
+            const Filter = ((rulePrefix && tags) || (tags && tags.length > 1)) ?
+                `<Filter><And>${Prefix}${tagXML}</And></Filter>` :
+                `<Filter>${Prefix}${tagXML}</Filter>`;
+
+            const Actions = actions.map(action => {
+                const { actionName, days, date, deleteMarker } = action;
+                let Action;
+                if (actionName === 'AbortIncompleteMultipartUploads') {
+                    Action = `<${actionName}><DaysAfterInitiation>${days}` +
+                        `</DaysAfterInitiation></${actionName}>`;
+                } else if (actionName === 'NoncurrentVersionExpiration') {
+                    Action = `<${actionName}><NoncurrentDays>${days}` +
+                        `</NoncurrentDays></${actionName}>`;
+                } else if (actionName === 'Expiration') {
+                    const Days = days ? `<Days>${days}</Days>` : '';
+                    const Date = date ? `<Date>${date}</Date>` : '';
+                    const DelMarker = deleteMarker ?
+                        `<ExpiredObjectDeleteMarker>${deleteMarker}` +
+                        '</ExpiredObjectDeleteMarker>' : '';
+                    Action = `<${actionName}>${Days}${Date}${DelMarker}` +
+                        `</${actionName}>`;
+                }
+                return Action;
+            }).join('');
+            return `<Rule>${ID}${Status}${Filter}${Actions}</Rule>`;
+        }).join('');
+        return '<?xml version="1.0" encoding="UTF-8"?>' +
+        '<LifecycleConfiguration ' +
+            'xmlns="http://s3.amazonaws.com/doc/2006-03-01/">' +
+            `${rulesXML}` +
```

Comment:
minor, but don't need the backticks and braces

Reply:
Removing them throws a linter error because the linter doesn't like string concatenation. That's why I left them in +1